auth/src/main/java/com/firebase/ui/auth/data/remote/EmailSignInHandler.java

```diff
@@ -29,7 +30,7 @@ public void startSignIn(@NonNull HelperActivityBase activity) {
 
     @Override
     public void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
-        if (requestCode == RequestCodes.EMAIL_FLOW) {
+        if (requestCode == RequestCodes.EMAIL_FLOW && resultCode != ErrorCodes.ANONYMOUS_UPGRADE_MERGE_CONFLICT) {
```

Comment:
nit: this may be a little clearer if you add a separate block at the top:

```
if (resultCode == ErrorCodes.ANONYMOUS_UPGRADE_MERGE_CONFLICT) {
  // Add comment explaining why  
  return;
}
```

Reply:
Done.